lib/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -34,7 +34,7 @@ export const useStyles = makeStyles(
       right: 8,
     },
   }),
-  { name: 'MuiPickerDTToolbar' }
+  { name: 'MuiPickersDateTimePickerToolbar' }
```

Comment:
```suggestion
  { name: 'MuiDateTimePickerToolbar' }
```

Reply:
If the prefix is really just `Mui` and not `MuiPickers`, that might cause clashes on things like the `ModalDialog`. Currently, that one is not exported, but I'm not sure if even `PrivateModalDialog` could clash with another material-ui-based library that has its own ModalDialog?